parsec/backend/postgresql/organization.py

```diff
@@ -272,7 +272,7 @@ async def create(
                     *_q_insert_organization(
                         organization_id=id.str,
                         bootstrap_token=bootstrap_token,
-                        active_users_limit=active_users_limit,
+                        active_users_limit=active_users_limit.to_int(),
```

Comment:
Should be handle by `postgres handler`

Reply:
This means that we need to set a custom type handlers for `integer`. Can this create problems with other fields?

Reply:
The handler must catch any `integer type`

Reply:
Or another solution is to change to `int | None` in `python world`, and this `enum` only exists in `rust world`

Reply:
But it's never good to add logic to data

Reply:
Don't forget to remove `.to_int()`